cmds/wifi/wifiServer.go

```diff
@@ -13,11 +13,11 @@ import (
 	"net/http"
 
```

Comment:
you're gonna kill me but ... could we make this name server.go instead of wifiServer.go?

Reply:
Oh, I plan on doing this in the next PR where I refactor and clean up the code in wifi

Reply:
Fixed